### PR TITLE
Fix apiserver deployment on Mirantis

### PR DESCRIPTION
**Pull request by freecaykes**
## Description
fix - calico/tigera-apiserver deployment for Mirantis kube-apiserver throwing exception of 
`time="2021-06-23T19:18:49Z" level=error msg="[DCT Signing Policy] error: deployments.apps \"tigera-apiserver\" is forbidden: unable to lookup image pull secret \"tigera-pull-secret\" from namespace \"tigera-system\": secrets \"tigera-pull-secret\" not found"`

because of docker content trust verifies the  calico/tigera-apiserver image we are pulling with the imagepullsecret  

Moving the order of  copying the tigera-pull-secret to the calico/tigera-apiserver namespace first before deploying  calico/tigera-apiserver.   

## For PR author

- [ ] Tests for change.
- [ ] If changing pkg/apis/, run `make gen-files`
- [ ] If changing versions, run `make gen-versions`

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [ ] Milestone set according to targeted release.
- [ ] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

**Review by doublek — APPROVED** [review]
LGTM

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**
